src/layout.js

```diff
@@ -461,7 +480,7 @@ export function applyStaticLayout(element) {
     // i-amphtml-sizer element
     const sizer = htmlFor(element)`
       <i-amphtml-sizer class="i-amphtml-sizer">
-        <img alt="" role="presentation" aria-hidden="true" 
```

Comment:
This happened automatically but it saves us exactly one byte in the binary.

![giphy-downsized-large](https://user-images.githubusercontent.com/254946/53056038-ae753d80-345e-11e9-8d37-f78bb6843d6d.gif)

Reply:
lol